service/index.js

```diff
@@ -1,27 +1,36 @@
 const Contact = require("./schemas/contacts");
+const Users = require("./schemas/users");
 
-const listContacts = async () => {
-  return Contact.find();
+const listContacts = async (userId) => {
+  return Contact.find({ owner: userId });
 };
 
-const getContactById = async (contactId) => {
-  return Contact.findOne({ _id: contactId });
+const getContactById = async (userId, contactId) => {
+  return Contact.findOne({ owner: userId, _id: contactId });
 };
 
-const addContact = async (contact) => {
-  return Contact.create(contact);
+const addContact = async (userId, contact) => {
+  return Contact.create({ owner: userId }, contact);
```

Comment:
```suggestion
  return Contact.create({...contact,  owner: userId });
```
Zobacz to sobie, do owner trzeba przypisać userId.

Reply:
https://mongoosejs.com/docs/api/model.html#Model.create()

Reply:
Done :-)